packages/gcp/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:alpine
+
+# ADD ./chromium-x64 /opt/bin/chromium
```

Comment:
Is it possible to pull Chromium from [lambda-binaries](https://github.com/remotion-dev/lambda-binaries/tree/main/chromium-x64)? At the moment I'm including the whole folder within the repo (on my machine at least)

Reply:
We could theoretically download it from Git, or we also have a S3 bucket which acts as a CDN already for our FFmpeg binaries.

For now it's fine to keep it directly locally